modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,385 @@
+//provides includes for reference sections in anypoint-insights.adoc 
+
+//-----------------------//
+// APP DETAIL HEADER INFO:
+//tag::app-detail-header[]
+Type::
+Mule application type (*Application*)
+
+Status::
+Status code that indicates whether this Mule app is running or in some other xref:runtime-manager::managing-deployed-applications.adoc#status_states[state]
++
+This code comes from Runtime Manager.
+
+URL:: 
+URL of this deployed Mule app
+
+//TODO: Runtime -> Runtime Version (SATHISH)
+Runtime::
+Mule runtime engine version of this Mule app
+
+Replicas::
+Virtual core fraction and count for this Mule app, for example, *0.1 vCores x 1* (= 10% of 1 vCore)
```

Comment:
Bold doesn't get translated. OK?

Reply:
That would be what people see in the UI. So bold is ok here.